server/controllers/user.controller.js

```diff
@@ -60,40 +61,37 @@ export const signup = async (req, res) => {
 };
 
 
-export const signin = (req, res) => {
-//   if (!req.body.email || !req.body.password)
-//     return res.status(400).send({
-//       status: 400,
-//       error: "please check your credentials"
-//     });
-
-//   const newUser = User.find(c => c.email == req.body.email);
-
-//   if (!newUser)
-//     return res.status(400).send({
-//       status: 400,
-//       error: "User with provided email or password do not exist!"
-//     });
-
-//   const token = jwt.sign({ email: newUser.email }, process.env.MY_SECRET, {
-//     expiresIn: "2d"
-//   });
+export const signin = async (req, res) => {
+    let { email, password } = req.body;
```

Comment:
Are you re-assigning these elements? Please, the scoping here. should be const instead

Reply:
I collected it